deploy/main.tf

```diff
@@ -25,8 +25,8 @@ resource "aws_ecs_task_definition" "insights-git-task" {
   task_role_arn = aws_iam_role.ecs_task_role.arn
   container_definitions = jsonencode([
     {
-      name      = "insights-git"
-      image     = "linonymous/insights-git-binary:latest"
+      name      = "insights-connector-git"
+      image     = "395594542180.dkr.ecr.us-east-1.amazonaws.com/insights-git-connector:latest"
```

Comment:
@linonymous image name should be `insights-connector-git` as per convention.
CC: @fayazg

Reply:
done.